system/Test/CIDatabaseTestCase.php

```diff
@@ -135,6 +142,8 @@ public function __construct()
 	 */
 	public function setUp()
 	{
+		$this->loadDependencies();
```

Comment:
No, I didn't mean for you to revert the commit. If your changes allow the rest of the tests to keep running and the live database tests just fail or error, then that's what I'm good with. That works better for the user.